src/app/screens/speedUpTransaction/index.tsx

```diff
@@ -128,7 +131,7 @@ function SpeedUpTransactionScreen() {
     } finally {
```

Comment:
I wonder if we actually want to start displaying some errors in toast for catch errors like these...

Reply:
makes sense! I added a todo comment for this